### ISSUE TITLE
[Bug] MacOS 10.13.x doesn't have _NSAppearanceNameAccessibilityHighContrastAqua

### ISSUE THREAD

**Issue by asudakov** [lencx/ChatGPT#253]
### Version

0.8.0 and later

### Bug description

dyld: Symbol not found: _NSAppearanceNameAccessibilityHighContrastAqua

Expected in: /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
 in /Users/zander/Downloads/ChatGPT.app/Contents/MacOS/ChatGPT

### OS

MacOS 10.13.x

**Comment by lencx** [comment]
It may be caused by the system not supporting the theme: Only supported on `macOS 10.14+`.
